### PR TITLE
doc/release-notes: Luminous release notes typo fixes  "ceph config-key ls"->"ceph config-key list"

### PR DESCRIPTION
**Pull request by scienceluo**
"ceph config-key ls" should be "ceph config-key list"

Signed-off-by: luo.kexue@zte.com.cn

### CODE REVIEW

**Comment by xiexingguo** [comment]
locally verified:

```
 ./bin/ceph config-key ls

no valid command found; 10 closest matches:
config-key rm <key>
config-key exists <key>
config-key list
config-key dump
config-key get <key>
config-key put <key> {<val>}
config-key del <key>
Error EINVAL: invalid command

./bin/ceph config-key list
[
    "mgr/dashboard/x/server_addr",
    "mgr/dashboard/x/server_port",
    "mgr/restful/keys/admin",
    "mgr/restful/x/crt",
    "mgr/restful/x/key",
    "mgr/restful/x/server_addr",
    "mgr/restful/x/server_port"
]
```

**Comment by xiexingguo** [comment]
Can you drop the chmod changes?

File: doc/release-notes.rst [hunk 1]
```diff
@@ -214,7 +214,7 @@ Major Changes from Kraken
       fails if clients older than the specified release are currently
       connected to the cluster.
     - ``ceph config-key dump`` dumps config-key entries and their
-      contents.  (The exist ``ceph config-key ls`` only dumps the key
+      contents.  (The exist ``ceph config-key list`` only dumps the key
```
Comment (tchaikov) [inline doc/release-notes.rst]: s/exist/existing/

Comment (scienceluo) [inline doc/release-notes.rst]: Done, thanks.

**Pull request merged**